README.md

```diff
@@ -1,19 +1,137 @@
 # elastalert-k8s-automation
-Use power of Kubernetes to automation configuration of elastaler
+Use power of Kubernetes to automation configuration of ElastAlert.
 
-## Building
-To build and upload image to the repository run the following commands:
+The goal of the current project is to simplify configuring ElastAlert notifications about events of some other 
+applications that store its logs in ElasticSearch.
+
+There are two modes for running this application: the local mode and the remote mode.
+* Remote mode is intended for running application in Kubernetes cluster where ElastAlert 
+is started.
+* Local mode allows to run application without deployment in Kubernetes. 
+
+The application uses two types of configuration files to configure work of ElastAlert:
+* Administrator configuration file with general properties and notification options.
+* User rules configuration files that determine rules for ElastAlert.
+
+The application collects both types of configuration files and generates configuration for ElastAlert.
+
+## Installing and configuring
+Clone elastlert-k8s-automation project:
```

Comment:
Actually for installation we need to just `kubectl apply -f <URL to file with deployment and config>`.
So, in this case we need to prepare docker image building and merge deployment and configmap to one file in `example` directory.
I will prepare docker image building, and you should create example.